query_builders/query_builder.py

```diff
@@ -804,7 +815,10 @@ def generate_return(self,
                 tag = label['tag']
             if return_nodeid:
                 id_col_name = self.gen_id_col_name(label, tag)
-                item_str = f"{{`{id_col_name}`:id(`{label}`)}}"
+                if label in labels_to_pack:
+                    item_str = f"{{`{id_col_name}`:`ids_{label}`}}"
+                else:
+                    item_str = f"{{`{id_col_name}`:id(`{label}`)}}"
```

Comment:
Here sometimes `labels_to_pack` is None, so we will need to check for this beforehand!